webdriver_manager/drivers/chrome.py

```diff
@@ -85,5 +107,20 @@ def get_url_for_version_and_platform(self, browser_version, platform):
                 for d in downloads:
                     if d["platform"] == platform:
                         return d["url"]
+        return None
 
-        raise Exception(f"No such driver version {browser_version} for {platform}")
+    def get_latest_release_for_version(self, version):
+        url = "https://googlechromelabs.github.io/chrome-for-testing/last-known-good-versions.json"
+        response = self._http_client.get(url)
+        data = response.json()
+        versions =[v["version"] for v in data["channels"].values()]
+        for v in versions:
+            if version in v:
+                return v 
+        return None
+
+    def get_latest_patch_version_for_build_version(self, build_version):
```

Comment:
@SergeyPirogov Here is the function for latest release by build version ie. 115.0.5735

Reply:
So, why do we need it?

Reply:
@SergeyPirogov The old LATEST_RELEASE url supported those style lookups. ie. https://chromedriver.storage.googleapis.com/LATEST_RELEASE_113 and https://chromedriver.storage.googleapis.com/LATEST_RELEASE_114.0.5735. [It was suggested we retain that for versions over 115](https://github.com/SergeyPirogov/webdriver_manager/pull/537#discussion_r1270761387). Without that, users will have to pass in the full browser version ie. version="114.0.5735.90", which can change frequently. Either way, I'm fine including for backwards compatibility, I'm fine with removing for simplicity and add an exception that explains as suggested by you

Reply:
I'd postpone this, because what I see know is that browser version is equal to the one mentioned in their driver website. Moreover I think that fix should be applied in method **get_url_for_version_and_platform**